docs/process/release.md

```diff
@@ -177,6 +177,15 @@ yarn # Reload dependencies
 yarn admin publish --tag next
 ```
 
+If also publish an LTS branch, check out that patch branch (e.g. `8.3.x`), then
+run:
+```bash
+yarn # Reload dependencies
+yarn admin publish --tag v8-lts
+```
+
+**Make sure to update the tag for the version you are releasing!**
```

Comment:
We should probably mention this before the release steps.
Also `tag` here is overloaded, since it could be git tag or npm tag.
It's best to be explicit.